src/libs/jsoncons/build_files/cmake/config.cmake.in

```diff
@@ -0,0 +1,13 @@
+# jsoncons cmake module
```

Comment:
Is this file needed? If we have the CMakeList, other makefiles should not be needed, as they are generated automatically.

Reply:
Hi @frisso 
I tried to modify the CMakeLists as little as possible and I left this file because it is used to set 3 variables. I haven't investigated that much at the time, but now taking a look it seems like deleting this file and deleting it
```
configure_package_config_file (build_files / cmake / config.cmake.in
                              "$ {CMAKE_CURRENT_BINARY_DIR} / $ {} PROJECT_NAME Config.cmake"
                              INSTALL_DESTINATION $ {JSONCONS_CMAKECONFIG_INSTALL_DIR})

```

from CMakeLists.txt don't give any errors, but I don't know if the variables have remained set somewhere.